gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 #Sun Oct 27 14:39:59 EDT 2019
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.5.1-all.zip
```

Comment:
Hmm, rolling back gradle isn't particularly a good idea. You should really seek alternatives like the shadow plugin from johnrengelman

Reply:
Indeed, I'm tweaking this now and will push shortly.